test/Interop/Cxx/static/Inputs/constexpr-static-member-var-errors.h

```diff
@@ -11,3 +11,11 @@ struct GetTypeValueInline {
 };
 
 using Invalid2 = GetTypeValueInline<int>;
+
+struct S {};
+template <class T>
+struct IsSubtypeSame {
+  static constexpr const S value = T();
+};
+
+using Invalid3 = IsSubtypeSame<int>;
```

Comment:
Maybe you can add another test where S is comparable and we validate that `IsSubtypeSame<S>::value == S()` (and that a constexpr is generated)?